packages/pnpm-sync-lib/src/pnpmSyncPrepare.ts

```diff
@@ -38,6 +40,7 @@ export interface IPnpmSyncPrepareOptions {
 export async function pnpmSyncPrepareAsync({
```

Comment:
Have you considered making a Jest test for this function?  Since it purely transforms text files and all I/O is abstracted through `IPnpmSyncPrepareOptions`, maybe it would be relatively easy to test.

Reply:
Good suggestion, I will add it in the separated PR